Library/Formula/languagetool.rb

```diff
@@ -18,4 +18,8 @@ def install
     (bin+"languagetool-server").write server_script(libexec/"languagetool-server.jar")
     bin.write_jar_script libexec/"languagetool.jar", "languagetool-gui"
   end
+
+  test do
+    system "echo 'This is a test.' | languagetool -l en-US -"
```

Comment:
Please use `pipe_output("#{bin}/languagetool -l en-US -", "This is a test.")` instead.